core/schemas/observables/iban.py

```diff
@@ -0,0 +1,10 @@
+from typing import Literal
+
+from core.schemas import observable
+
+
+class Iban(observable.Observable):
```

Comment:
```suggestion
class IBAN(observable.Observable):
```

We use uppercase for other observables made of acronyms such as CIDR. (granted, some are not compliant with this, we might change it in the future)